source/whats-new.txt

```diff
@@ -50,6 +50,9 @@ The v4.1 driver release includes the following new features:
   - :manual:`Binary Data BSON Types </reference/bson-types/#binary-data>` in the {+mdb-server+} manual
   - :github:`BSON Binary Vector </mongodb/mongo-cxx-driver/blob/master/examples/bsoncxx/bson_binary_vector.cpp>` usage example on GitHub
 - Fixes the :ref:`C driver API version bug <v4.0-cmake-api-version-bug>` noted in the v4.0 release
+- Adds a change to the ``ENABLE_TESTS`` configuration default option that
```

Comment:
Q: is this bug fix a breaking change? Maybe verify with dbx

Reply:
This is both a bugfix (fixing behavior introduced in 4.0) and a 4.1 build system breaking change (for auto-downloaded C Driver configurations).